commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommercePriceEntryServiceImpl.java

```diff
@@ -119,70 +143,52 @@ public CommercePriceEntry fetchCommercePriceEntry(long commercePriceEntryId)
 			OrderByComparator<CommercePriceEntry> orderByComparator)
 		throws PortalException {
 
-		PortalPermissionUtil.check(
-			getPermissionChecker(),
+		_checkCommerceCatalogPermissionByCommercePriceListId(
+			commercePriceListId,
 			CommercePriceListActionKeys.MANAGE_COMMERCE_PRICE_LISTS);
 
 		return commercePriceEntryLocalService.getCommercePriceEntries(
 			commercePriceListId, start, end, orderByComparator);
 	}
 
-	@Override
-	public List<CommercePriceEntry> getCommercePriceEntriesByCompanyId(
-			long companyId, int start, int end)
-		throws PortalException {
-
-		PortalPermissionUtil.check(
-			getPermissionChecker(),
-			CommercePriceListActionKeys.MANAGE_COMMERCE_PRICE_LISTS);
-
-		return commercePriceEntryLocalService.
-			getCommercePriceEntriesByCompanyId(companyId, start, end);
-	}
-
 	@Override
 	public int getCommercePriceEntriesCount(long commercePriceListId)
 		throws PortalException {
 
-		PortalPermissionUtil.check(
-			getPermissionChecker(),
+		_checkCommerceCatalogPermissionByCommercePriceListId(
+			commercePriceListId,
 			CommercePriceListActionKeys.MANAGE_COMMERCE_PRICE_LISTS);
 
 		return commercePriceEntryLocalService.getCommercePriceEntriesCount(
 			commercePriceListId);
 	}
 
-	@Override
-	public int getCommercePriceEntriesCountByCompanyId(long companyId)
-		throws PortalException {
-
-		PortalPermissionUtil.check(
-			getPermissionChecker(),
-			CommercePriceListActionKeys.MANAGE_COMMERCE_PRICE_LISTS);
-
-		return commercePriceEntryLocalService.
-			getCommercePriceEntriesCountByCompanyId(companyId);
-	}
-
```

Comment:
Why remove this method?

Reply:
Because it is unused

Reply:
It may be unused by us -- but what about our community/customers? Unless there's a specific reason to remove it (e.g. we've removed the db index or something), it's much safer to just leave it in.